lib/ansible/module_utils/common/collections.py

```diff
@@ -0,0 +1,53 @@
+# Copyright (c), Sviatoslav Sydorenko <ssydoren@redhat.com> 2018
+# Simplified BSD License (see licenses/simplified_bsd.txt or https://opensource.org/licenses/BSD-2-Clause)
+"""Collection of low-level utility functions."""
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+try:
+    """Python 3.3+ branch."""
+    from collections.abc import (
+        deque, KeysView,
+        Mapping, MutableMapping,
+        Sequence, MutableSequence,
+        Set, MutableSet,
+    )
+except ImportError:
+    """Use old lib location under 2.6-3.2."""
+    from collections import (
+        deque, KeysView,
+        Mapping, MutableMapping,
+        Sequence, MutableSequence,
+        Set, MutableSet,
+    )
+
+
+from ansible.module_utils.six import binary_type, text_type
+
+
+def is_string_type(seq):
+    """Identify whether the input has a string-like type (inclding bytes)."""
+    return isinstance(seq, (text_type, binary_type))
```

Comment:
The other functions we have here don't end with _type so I'd name this without the _type as well:

``` python
def is_string(obj):
```